frontend/src/components/common/NullScreen.js

```diff
@@ -81,6 +82,13 @@ const NullScreen = (props) => (
         <p>{String.NO_PREV_MINE_MANAGER}</p>
       </div>
     )}
+    {props.type === "contacts" && (
+      <div>
+        <img src={MINER_TWO} />
+        <h3>No contacts found</h3>
+        <p>Create a contact using the menu above</p>
```

Comment:
Create constants for these, reuse existing strings when necessary to keep `Null screens` consistent.

Reply:
Yeah, you are correct in that.

I guess my point was to reuse as much as possible in terms of the strings so the null screens are consistent as possible with their verbiage. 

I'm happy to remove the constants within this file and show the actual strings.